### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/IoT-For-Beginners/commit/152f85d8f8ced0c5b63649a5e9fb622ee2d36705#commitcomment-86437888

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/IoT-For-Beginners/commit/ce19082ff0ebc1299ff77884dd27276a2ea84af8
 
Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: 2-farm/lessons/5-migrate-application-to-the-cloud/README.md [hunk 1]
```diff
@@ -375,7 +375,7 @@ This will create a folder inside the `soil-moisture-trigger` folder called `iot-
     For detailed output, run func with --verbose flag.
     [2021-05-05T02:44:07.517Z] Worker process started and initialized.
     [2021-05-05T02:44:09.202Z] Executing 'Functions.iot-hub-trigger' (Reason='(null)', Id=802803a5-eae9-4401-a1f4-176631456ce4)
-    [2021-05-05T02:44:09.205Z] Trigger Details: PartionId: 0, Offset: 1011240-1011632, EnqueueTimeUtc: 2021-05-04T19:04:04.2030000Z-2021-05-04T19:04:04.3900000Z, SequenceNumber: 2546-2547, Count: 2
+    [2021-05-05T02:44:09.205Z] Trigger Details: PartitionId: 0, Offset: 1011240-1011632, EnqueueTimeUtc: 2021-05-04T19:04:04.2030000Z-2021-05-04T19:04:04.3900000Z, SequenceNumber: 2546-2547, Count: 2
```
Comment (jsoref) [inline 2-farm/lessons/5-migrate-application-to-the-cloud/README.md:378]: It's possible this is wrong. Did this output come from buggy external code?

Comment (jimbobbennett) [reply 2-farm/lessons/5-migrate-application-to-the-cloud/README.md:378]: Yeah - this is the output of the Azure Functions event hub connector. Probably ok to have a spelling fix, and I'm trying to find the relevant code to get that fixed

**Comment by jimbobbennett** [comment]
Thank you @jsoref! Appreciate that this isn't just a run a spell check and raise a PR, that a review has been done to check the corrections.

**Review by jimbobbennett — APPROVED** [review]

**Comment by jsoref** [comment]
Yeah, my tooling can only identify candidates and offer suggestions (the actually spelling suggestions, as noted, don't even come from my tool, just a chained tool), it still requires a human to consider and select suggestions. It's a combination of mental exercise for me as well as a way to shake out bugs in my tool (this week's run has helped pushed forward a handful of in progress features).

Thanks for looking for the source of that item. I have a list of repositories to check, but it isn't prioritized and is incredibly long.

**Pull request merged**